release-notes/testing.yml

```diff
@@ -1,4 +1,7 @@
 releases:
+  38.20231002.2.2:
+      - text: "CVE-2023-4911: Buffer overflow in glibc leading to privilege escalation"
+        url: "https://github.com/coreos/fedora-coreos-tracker/issues/1591"
```

Comment:
```suggestion
  38.20231002.2.2:
    issues:
      - text: "CVE-2023-4911: Buffer overflow in glibc leading to privilege escalation"
        url: "https://github.com/coreos/fedora-coreos-tracker/issues/1591"
```

Reply:
`issues:` wasnt included in the last entry either, https://github.com/coreos/fedora-coreos-streams/pull/773/files, and I'm not seeing a release notes entry for `38.20230918.2.0` on the download page.